### PR TITLE
tests are passed

### PR DESCRIPTION
**Pull request by kubranarci**
## PR checklist

Closes #XXX 

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [x] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [x] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [x] Follow the naming conventions.
- [x] Follow the parameters requirements.
- [x] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [ ] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - [x] `PROFILE=docker pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=singularity pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`
  - [ ] `PROFILE=conda pytest --tag <MODULE> --symlink --keep-workflow-wd --git-aware`

### CODE REVIEW

File: modules/nf-core/truvari/segment/tests/main.nf.test
Comment (vagkaratzas) [inline modules/nf-core/truvari/segment/tests/main.nf.test:35]: align properly here

**Review by vagkaratzas — COMMENTED** [review]
Seems good! Would wait for an approval from @nvnieuwk though since he has already created the truvari/bench module.

File: modules/nf-core/truvari/segment/tests/main.nf.test
Comment (nvnieuwk) [inline modules/nf-core/truvari/segment/tests/main.nf.test:1]: Can you also add a stub test?

**Review by nvnieuwk — COMMENTED** [review]
Looking good! I'll approve if my comment is resolved :rocket:

File: modules/nf-core/truvari/segment/tests/main.nf.test
Comment (kubranarci) [reply modules/nf-core/truvari/segment/tests/main.nf.test:1]: okay

**Review by nvnieuwk — APPROVED** [review]
LGTM!

**Pull request merged**
